src/main/java/uk/gov/hmcts/ccd/domain/model/common/DisplayContextParameterUtil.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.hmcts.ccd.domain.model.common;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.apache.commons.lang3.StringUtils.isBlank;
+import static uk.gov.hmcts.ccd.domain.model.common.DisplayContextParameterCollectionOptions.ALLOW_DELETE;
+import static uk.gov.hmcts.ccd.domain.model.common.DisplayContextParameterCollectionOptions.ALLOW_INSERT;
+
+public class DisplayContextParameterUtil {
+
+    public static String COLLECTION = "COLLECTION";
+
+    public static String updateCollectionDisplayContextParameter(final String currentDisplayContextParameter,
```

Comment:
this code could seems a bit too complex would be good if we could further simplify by breaking it down into more private method with self explanatory names

Reply:
Will try to make it more clear.